.svgrrc

```diff
@@ -3,5 +3,6 @@
 		"#00F": "{props.stroke}",
 		"#000": "{props.fill}",
 		"#f00": "{props.fillSecondary}"
-	}
+	},
+	"icon": 24
```

Comment:
What this is set to seems arbitrary and ignored, but set to 24 as our standard icon size just in case since it otherwise defaults to "1em" which is more of a web thing. Internally, sets height/width in the SVG to "24" within the repo (does not alter the SVG files themselves), but basically seems to let icons scale freely.